### DISCUSSION TITLE
Incorrect Order of Files

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7775] [Help]
### ImageMagick version

7.1.1-40

### Operating system, version and so on

Windows 11

### Description

I have this command to make an APNG out of an image sequence but the files were in the wrong order.
`magick *.png apng:output.apng`

The png files are named input1-input178, like input%d.png. How do I fix this?

**Comment by fmw42** [comment]
Images are accessed in alphabetic order.  Use leading zeroes in your file names such as input001 ... image178. Get an app to rename files to add leading zeroes.

**Reply by Jay123210599** [reply]
How do I make be accessed in numerical order, instead?

**Reply by fmw42** [reply]
You cannot. You need to edit your filenames to add leading zeroes as mentioned above.  You can fine apps that will do that on your computer.

**Reply by fmw42** [reply]
Perhaps you might find some feature in ffmpeg.  I do not know if that is possible there, but you cannot from Imagemagick.

**Comment by snibgo** [comment]
There may be an app that can do this, but a simple script can loop through the files, renaming each one.
